### PR TITLE
Fix broken links - 2024.01

### PR DESCRIPTION
**Pull request by n-boshnakov**
**How to categorize this PR?**

/area documentation
/kind enhancement

**What this PR does / why we need it**:
This PR fixes broken links in the repo.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```doc developer
NONE
```

### CODE REVIEW

**Review by ary1992 — COMMENTED** [review]
Thanks for the changes.
/lgtm

File: docs/extensions/logging-and-monitoring.md [hunk 1]
```diff
@@ -156,5 +156,4 @@ If the type of logs exposed in the Plutono instances needs to be changed, it is
 
 * [GitHub Issue Describing the Concept](https://github.com/gardener/gardener/issues/1351)
 * [Exemplary Implementation (Monitoring) for the GCP Provider](https://github.com/gardener/gardener-extension-provider-gcp/blob/master/charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/configmap-observability.yaml)
-* [Exemplary Implementation (ClusterFilter) for the AWS Provider](https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/gardener-extension-provider-aws/templates/clusterfilters-logging.yaml)
-* [Exemplary Implementation (ClusterParser) for the AWS Provider](https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/gardener-extension-provider-aws/templates/clusterparsers-logging.yaml)
\ No newline at end of file
+* [Exemplary Implementation (ClusterFilter) for the AWS Provider](https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/gardener-extension-provider-aws/templates/clusterfilters-logging.yaml)
```
Comment (ialidzhikov) [inline docs/extensions/logging-and-monitoring.md]:
```suggestion
* [Exemplary Implementation (ClusterFilter) for the AWS Provider](https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/gardener-extension-provider-aws/templates/clusterfilters-logging.yaml)
* [Exemplary Implementation (ClusterParser) for the Shoot DNS Service](https://github.com/gardener/gardener-extension-shoot-dns-service/blob/master/charts/gardener-extension-shoot-dns-service/templates/clusterparsers-logging.yaml)
```

**Review by ialidzhikov — CHANGES_REQUESTED** [review]

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**